feat documentation authorization (#466)
fastify/fastify-swagger: README.md

- Pass `{ hide: true }` to the schema object inside the route declaration.
- Use the tag declared in `hiddenTag` options property inside the route declaration. Default is `X-HIDDEN`.

<a name="route.uiHooks"></a>
#### Protect your documentation routes

> You can protect your documentation configuring an authentication hook.

Reviewer: ```suggestion
You can protect your documentation by configuring an authentication hook.
```

> Here is an example with the [`fastify-basic-auth`](https://github.com/fastify/fastify-basic-auth) plugin:

Reviewer: ```suggestion
Here is an example using [`fastify-basic-auth`](https://github.com/fastify/fastify-basic-auth) plugin:
```

> Here is an example using [`fastify-basic-auth`](https://github.com/fastify/fastify-basic-auth) plugin:

Reviewer 2: ```suggestion
Here is an example using the [`fastify-basic-auth`](https://github.com/fastify/fastify-basic-auth) plugin:
```
